official/recommendation/ncf_test.py

```diff
@@ -215,7 +216,24 @@ def test_end_to_end_keras_mlperf(self):
          '-distribution_strategy', 'off',
          '-log_steps', '100'])
 
+  @mock.patch.object(rconst, "SYNTHETIC_BATCHES_PER_EPOCH", 100)
+  def test_end_to_end_keras_1_gpu(self):
+    integration.run_synthetic(
+        ncf_keras_main.main, tmp_root=self.get_temp_dir(), max_train=None,
+        extra_flags=self._BASE_END_TO_END_FLAGS +
+        ['-num_gpus', '1', '-log_steps', '100'])
```

Comment:
what's the log_steps used for ?

Reply:
It is used for the callback that is useful for benchmark. 
I don't think it is useful here.
Deleted.